samples/docker-domain/simple-topology.yaml

```diff
@@ -1,3 +1,6 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
These domain YAML files do NOT support comments or blank lines.

Reply:
Removed comments and blank lines.